### PR TITLE
Fix: Arrow press callbacks on expandable calendar

### PR DESCRIPTION
**Pull request by cjhines**
Resolve https://github.com/wix/react-native-calendars/issues/1488.

This is a release blocking issue for us sadly, so any feedback would be hugely appreciated.

As prop drilling is not happening for `onPressArrowLeft` and `onPressArrowRight` in  the `ExpandableCalendar` component, it should be included in the custom function which are being passed.

As a result, it is not possible to use these props in the `ExpandableCalendar` component. They just aren't called.

### CODE REVIEW

**Comment by cjhines** [comment]
https://github.com/wix/react-native-calendars/issues/1488

**Comment by cjhines** [comment]
I created a new PR without the linting issues. Please let me know if there are any further issues.

**Comment by cjhines** [comment]
E2E tests seem to be failing on master, as well as all PRs.

**Comment by cjhines** [comment]
Merged E2E fix into this branch. Checks are now passing.

**Comment by cjhines** [comment]
Would really appreciate that this gets looked at. This bug is currently a blocker for me.

File: src/expandableCalendar/index.js [hunk 1]
```diff
@@ -350,10 +350,18 @@ class ExpandableCalendar extends Component {
   /** Events */
 
   onPressArrowLeft = () => {
+    const {onPressArrowLeft} = this.props;
+    if (onPressArrowLeft) {
```
Comment (Inbal-Tish) [inline src/expandableCalendar/index.js:354]: Please use `_.invoke` instead of if statement

File: src/expandableCalendar/index.js [hunk 2]
```diff
@@ -350,10 +350,18 @@ class ExpandableCalendar extends Component {
...
+      onPressArrowLeft();
+    }
     this.scrollPage(false);
   };
 
   onPressArrowRight = () => {
+    const {onPressArrowLeft} = this.props;
+    if (onPressArrowLeft) {
```
Comment (Inbal-Tish) [inline src/expandableCalendar/index.js:362]: Please use `_.invoke` instead of if statement

File: src/expandableCalendar/index.js (hunk 1 above)
Comment (cjhines) [reply src/expandableCalendar/index.js]: @Inbal-Tish Done.

File: src/expandableCalendar/index.js (hunk 2 above)
Comment (cjhines) [reply src/expandableCalendar/index.js]: @Inbal-Tish Done.

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**